src/extras/reload.js

```diff
@@ -0,0 +1,69 @@
+(function (global) {
+  global.TRACING = true;
+
+  const systemJSPrototype = System.constructor.prototype;
+  const dependentsForModule = {};
+
+  systemJSPrototype.reload = function (id, parentUrl) {
```

Comment:
If the id is cached in the browser, I wonder if it is worth supporting a query parameter argument?

Something like `function (id, parentUrl, queryParam?)`.

Reply:
I'm not sure about the use case. If the web server is using etags based on last-modified, it should be fine. 

If the web server is using time based caching regular reloads wouldn't bust the cache either so that seems very impractical?

Reply:
Let's leave it for now, but lots of users ask for this, so I'm sure it will come up again soon enough.